lib/suse/connect/remote/product.rb

```diff
@@ -13,6 +13,7 @@ def initialize(product_hash)
     super
     # TODO: ensure we have array here
     self.extensions = extensions.map { |ext| self.class.new(ext) } if extensions
+    self.isbase ||= (product_hash['product_type'] == 'base')
```

Comment:
when we fetch activated products, we have to assign `isbase` attribute as well because otherwise they are considered as different products to `#installed_products` and we get `Multiple base products found` error
see https://github.com/SUSE/connect/blob/master/lib/suse/connect/status.rb#L57